### PR TITLE
[WIP] Pm/device auth mode x509

### PR DESCRIPTION
**Pull request by PatrikMatiasko**

### CODE REVIEW

File: web/src/config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  appName: 'plgd Dashboard',
-  defaultLanguage: 'en',
-  supportedLanguages: ['en'],
+    appName: 'plgd Dashboard',
```
Comment (ondrejtomcik) [inline web/src/config.js:2]: plgd Client

File: web/src/containers/App/App.tsx [hunk 2]
```diff
@@ -1,99 +1,84 @@
...
+        })
+    }
+
+    if (!wellKnownConfig) {
+        return <AppLoader />
+    }
+
+    const oidcCommonSettings = {
+        authority: wellKnownConfig?.remoteProvisioning.authorization.authority || '',
+        scope: wellKnownConfig?.remoteProvisioning.authorization.scopes.join?.(' ') || 'openid',
```
Comment (ondrejtomcik) [inline web/src/containers/App/App.tsx:47]: gotcha!

File: web/src/containers/App/AppContext.types.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { BuildInformationType } from '@/containers/App/App.types'
 
 export type AppContextType = {
-  buildInformation?: BuildInformationType | null
-  collapsed: boolean
-  httpGatewayAddress: string
+    buildInformation?: BuildInformationType | null
+    collapsed: boolean
+    setInitializedByAnother?: () => void
```
Comment (ondrejtomcik) [inline web/src/containers/App/AppContext.types.ts:6]: If I may, I would propose: `alreadyInitializedByOther` :)

**Review by ondrejtomcik — APPROVED** [review]

**Comment by jkralik** [comment]
Merge manually to branch `jkralik/feature/initializatioViaUserAgent`

**Pull request closed without merging**